app/views/snacks/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1><%= @snack.name %></h1>
+<p>Price: $<%= @snack.price %></>
+<p>Locations</p>
+<% @snack.machines.each do |machine| %>
+  <p><%=machine.location%> (<%=machine.snacks.length%> kinds of snacks, average price of $<%=machine.average%>)</p>
```

Comment:
machine.snacks.length is very quick logic, but ideally, we'd pull that out to a model method.